selftests/functional/test_utils.py

```diff
@@ -155,5 +160,35 @@ def test_process_run(self):
     def tearDown(self):
         shutil.rmtree(self.base_logdir)
 
+
+def file_lock_action(args):
+    path, players = args
+    max_individual_timeout = 1
+    max_timeout = max_individual_timeout * players
+    with FileLock(path, max_timeout) as lock:
+        sleeptime = random.random()
```

Comment:
Basically I'm happy with anything between `/10 - /100`. Less would be IMO too short, more is IMO not enough....